build/includes/release.mk

```diff
@@ -95,6 +95,11 @@ do-release: $(ensure-build-image)
 
 	@echo "Now go make the $(RELEASE_VERSION) release on Github!"
 
-build-release: 
-	cd $(agones_path) && gcloud builds submit . --substitutions _VERSION=$(base_version) --config=./build/release/cloudbuild.yaml $(ARGS)
+ifdef RC_RELEASE
```

Comment:
🤔  somehow this got added back in. I don't think it was supposed to.

Reply:
Good catch, I think that was part of the rebase merge conflict, I'll update.

Reply:
Other than this, this good to merge 👍🏻 Just checked the other formatting comments below.